src/components/dar_collection_table/DarCollectionTable.js

```diff
@@ -27,147 +28,86 @@ const styles = {
     alignItems: 'center'
   },
   columnStyle: Object.assign({}, Styles.TABLE.HEADER_ROW, {
-    justifyContent: 'space-between'
+    justifyContent: 'space-between',
+    color: '#7B7B7B',
+    fontFamily: 'Montserrat',
+    fontSize: '1.2rem',
+    fontWeight: 'bold',
+    letterSpacing: '0.2px',
+    textTransform: 'uppercase'
   }),
   cellWidth: {
     darCode: '15%',
-    projectTitle: '25%',
-    submissionDate: '15%',
-    status: '15%',
-    actions: '20%'
-  }
+    projectTitle: '20%',
+    submissionDate: '12.5%',
+    pi: '5%',
+    institution: '20%',
+    datasetCount: '7.5%',
+    status: '10%',
+    actions: '7.5%'
```

Comment:
Since we're changing the width of the actions column we need to make sure it doesn't break the styling elsewhere. Easy way to do this is to make your own dar locally and then visit the `/new_researcher_console` route and see the changes.

Reply:
If there's issues, mess around with the values. It's going to primarily be trial and error here.